### PR TITLE
3.21 Release Post

### PR DESCRIPTION
**Pull request by kennethlarsen**

### CODE REVIEW

File: source/2020-09-01-ember-3-21-released.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+npx ember-cli-update
+```
+
+This utility will help you to update your app or addon to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the github README](https://github.com/ember-cli/ember-cli-update).
+
+While it is recommended to keep Ember CLI versions in sync with Ember and Ember Data, this is not required. After updating ember-cli, you can keep your current version(s) of Ember or Ember Data by editing `package.json` to revert the changes to the lines containing `ember-source` and `ember-data`.
+
+### Changes in Ember CLI 3.21
+
+#### New Features (?)
```
Comment (rwjblue) [inline source/2020-09-01-ember-3-21-released.md]: `ember new --lang` support landed in https://github.com/ember-cli/ember-cli/pull/9259

Comment (rwjblue) [reply source/2020-09-01-ember-3-21-released.md]: Tweak ember-try configuration for addons:

* drop Ember 3.12 (existing addons should consider this a breaking change) https://github.com/ember-cli/ember-cli/pull/9310
* add Ember 3.20 https://github.com/ember-cli/ember-cli/pull/9308

Comment (ijlee2) [reply source/2020-09-01-ember-3-21-released.md]: Thanks! I will make sure to mention these items.

File: source/2020-09-01-ember-3-21-released.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.21
+
+Ember Data 3.21 release saw a few improvements to documentation and continuous integration.
```
Comment (ijlee2) [inline source/2020-09-01-ember-3-21-released.md]: In the [changelog](https://github.com/emberjs/data/blob/v3.21.0/CHANGELOG.md), I saw that [ember-data#7228](https://github.com/emberjs/data/pull/7228). I wasn't sure what the PR change did, however. If the change is important for developers to know, will you let me know how to explain it? Thanks.

File: source/2020-09-01-ember-3-21-released.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+### Upgrading Ember CLI
+
+<!--alex ignore easy-->
+You may upgrade Ember CLI easily using the ember-cli-update project:
+
+```bash
+npx ember-cli-update
+```
+
+This utility will help you to update your app or addon to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the github README](https://github.com/ember-cli/ember-cli-update).
```
Comment (amyrlam) [inline source/2020-09-01-ember-3-21-released.md]:
```suggestion
This utility will help you to update your app or addon to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the GitHub README](https://github.com/ember-cli/ember-cli-update).
```

**Review by amyrlam — APPROVED** [review]
looks good, left one suggestion (non-blocking), remove WIP from PR title before merge?

**Comment by ijlee2** [comment]
@amyrlam Thanks for reviewing the post!

@jenweber and @mansona will work on publishing the blog post, along with a new version of the Ember Guides.

File: source/2020-09-01-ember-3-21-released.md (hunk 2 above)
Comment (ijlee2) [reply source/2020-09-01-ember-3-21-released.md]: @igorT @runspired If you have time, can you help me with answering this question?

File: source/2020-09-01-ember-3-21-released.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.21
+
+Ember.js 3.21 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+#### New Features
```
Comment (jaredgalanis) [inline source/2020-09-01-ember-3-21-released.md]: Not a blocker at all, but it looks to me like we have typically added the number of features/deprecations in parens in the heading, even if it's zero. Maybe we decided to move away from this though?
```suggestion
#### New Features (0)
```

**Review by jaredgalanis — APPROVED** [review]
One small comment. Not a blocker. Looks great, thank you for doing this!

File: source/2020-09-01-ember-3-21-released.md (hunk 4 above)
Comment (ijlee2) [reply source/2020-09-01-ember-3-21-released.md]: Yeah, I removed the numbers from the headings. If showing the numbers in parentheses was a convention, I'm okay with restoring the parentheses.

Comment (amyrlam) [reply source/2020-09-01-ember-3-21-released.md]: FWIW, historically there has been a lot of controversy over the numbering in #core-meta (sometimes the # count was wrong, or the # didn't seem that helpful), seems good to get rid of it 👍 *but no strong opinion!

File: source/2020-09-01-ember-3-21-released.md (hunk 3 above)
Comment (ijlee2) [reply source/2020-09-01-ember-3-21-released.md]: Thanks! I think the casing came from copy-paste. I fixed it in a commit.

**Pull request merged**